MovieQuiz/Presentation/MovieQuizViewController.swift

```diff
@@ -1,9 +1,169 @@
+        ),
+        QuizQuestion(
+            image: "The Avengers",
+            text: "Рейтинг этого фильма больше чем 6?",
+            correctAnswer: true
+        ),
+        QuizQuestion(
+            image: "Deadpool",
+            text: "Рейтинг этого фильма больше чем 6?",
+            correctAnswer: true
+        ),
+        QuizQuestion(
+            image: "The Green Knight",
+            text: "Рейтинг этого фильма больше чем 6?",
+            correctAnswer: true
+        ),
+        QuizQuestion(
+            image: "Old",
+            text: "Рейтинг этого фильма больше чем 6?",
+            correctAnswer: false
+        ),
+        QuizQuestion(
+            image: "The Ice Age Adventures of Buck Wild",
+            text: "Рейтинг этого фильма больше чем 6?",
+            correctAnswer: false
+        ),
+        QuizQuestion(
+            image: "Tesla",
+            text: "Рейтинг этого фильма больше чем 6?",
+            correctAnswer: false
+        ),
+        QuizQuestion(
+            image: "Vivarium",
+            text: "Рейтинг этого фильма больше чем 6?",
+            correctAnswer: false
+        )
+    ]
+    private var currentQuestionIndex: Int = 0
+    private var correctAnswers: Int = 0
+    
     // MARK: - Lifecycle
     override func viewDidLoad() {
         super.viewDidLoad()
+        imageView.layer.cornerRadius = 20
+        
+        show(quiz: convert(model: questions[currentQuestionIndex]))
+    }
+    
+    @IBAction private func yesButtonClicked(_ sender: UIButton) {
+        checkResponse(sender)
+    }
+
+    @IBAction private func noButtonClicked(_ sender: UIButton) {
+        checkResponse(sender)
+    }
+    
+    private func checkResponse(_ button: UIButton) {
+        if button.currentTitle == "Нет" {
+            let userResponse = QuizResultResponseViewModel(isCorrect: questions[currentQuestionIndex].correctAnswer ? false : true).isCorrect
+            showAnswerResult(isCorrect: userResponse)
+        } else {
+            let userResponse = QuizResultResponseViewModel(isCorrect: questions[currentQuestionIndex].correctAnswer ? true : false).isCorrect
+            showAnswerResult(isCorrect: userResponse)
+        }
+    }
+    
+    private func show(quiz step: QuizStepViewModel) {
+        imageView.image = UIImage(named: questions[currentQuestionIndex].image)
+        textLabel.text = "\(questions[currentQuestionIndex].text)"
+        counterLabel.text = "\(currentQuestionIndex + 1)/\(questions.count)"
+    }
+
+    private func show(quiz result: QuizResultsViewModel) {
+        let alert = UIAlertController(
+            title: result.title,
+            message: result.text,
+            preferredStyle: .alert)
+        let action = UIAlertAction(title: "Сыграть ещё раз", style: .default) { _ in
+            self.currentQuestionIndex = 0
+            self.correctAnswers = 0
+            self.show(quiz: self.convert(model: self.questions[self.currentQuestionIndex]))
+        }
```

Comment:
В ближайших спринтах вам расскажут про замыкания и про слабый список захвата, сейчас же предлагаю добавить `[weak self]`. Тогда придется обращаться к методам через `self?`. Также предложу вынести содержимое в отдельную функцию

```suggestion
        let action = UIAlertAction(title: "Сыграть ещё раз", style: .default) { [weak self] _ in
            self?.restartQuiz()
        }
```